src/main/resources/data-repository-openapi.yaml

```diff
@@ -1872,16 +1872,49 @@ definitions:
     properties:
       name:
         $ref: '#/definitions/ObjectNameProperty'
-
       columns:
         type: array
         items:
           $ref: '#/definitions/ColumnModel'
-
       primaryKey:
         type: array
         items:
           $ref: '#/definitions/ObjectNameProperty'
+      partitionStrategy:
+        $ref: '#/definitions/PartitionStrategy'
+      partitionColumn:
+        type: string
+        description: >
+          Name of the column to partition on, required if using the
+          'date_or_time_column' or 'int_column' partition strategy
+      intPartitionOptions:
+        $ref: '#/definitions/IntPartitionOptionsModel'
+        description: Extra settings for the 'int_column' strategy
+
+  PartitionStrategy:
+    type: string
+    enum:
+      - ingest_time
+      - date_or_timestamp_column
```

Comment:
Our convention is to use camelCase inside of the openapi yaml (except for the ga4gh endpoints and apparently we left it in for `created_date` in the `sort` enums..

Reply:
There are a few other spots (`array_of` is one I can think of off the top of my head). I'll update these, should I make a ticket for correcting the other inconsistencies?